examples/resources/astro_workspace/resource.tf

```diff
@@ -3,3 +3,14 @@ resource "astro_workspace" "example" {
   description           = "my first workspace"
   cicd_enforced_default = true
 }
+
+// Import an existing workspace
+import = {
+  id = "clozc036j01to01jrlgvu798d" // CUID of the existing workspace
```

Comment:
CUID is probably overkill to mention, we should just say ID since CUID is a protocol.

Reply:
true, ill change that